README.md

```diff
@@ -73,502 +58,177 @@ Native packages for your operating system will be provided in the near future.
-  -e ENABLE_CATALOG=True \
-  -e GRAPH_FACTS='architecture,puppetversion,osfamily' \
-  puppetboard
-```
+ ``` {.sourceCode .bash}
+ $ docker build -t puppetboard .
 
-To set a URL prefix you can use the optional `PUPPETBOARD_URL_PREFIX`
-environment variable.
+ $ docker run -it -p 9080:80 -v /etc/puppetlabs/puppet/ssl:/etc/puppetlabs/puppet/ssl \
+   -e PUPPETDB_HOST=<hostname> \
+   -e PUPPETDB_PORT=8081 \
+   -e PUPPETDB_SSL_VERIFY=/etc/puppetlabs/puppetdb/ssl/ca.pem \
+   -e PUPPETDB_KEY=/etc/puppetlabs/puppetdb/ssl/private.pem \
+   -e PUPPETDB_CERT=/etc/puppetlabs/puppetdb/ssl/public.pem \
+   -e INVENTORY_FACTS='Hostname,fqdn, IP Address,ipaddress' \
+   -e ENABLE_CATALOG=True \
+   -e GRAPH_FACTS='architecture,puppetversion,osfamily' \
+   puppetboard
+```
 
-### Development
+### Manually
 
-If you wish to hack on Puppetboard you should fork/clone the Github repository and then install the requirements through:
+To install it simply issue the following command:
 
 ``` {.sourceCode .bash}
-$ pip install -r requirements-test.txt
+$ pip install puppetboard
 ```
 
-You're advised to do this inside a virtualenv specifically created to work on Puppetboard as to not pollute your global Python installation.
+This will install Puppetboard and take care of the dependencies. If you do this Puppetboard will be installed in the so 
+called site-packages or dist-packages of your Python distribution.
 
-Configuration
--------------
+The complete path on Debian and Ubuntu systems would be `/usr/local/lib/pythonX.Y/lib/dist-packages/puppetboard`
+and on Fedora would be `/usr/lib/pythonX.Y/site-packages/puppetboard` where `X` and `Y` are replaced by your major
+and minor python versions.
 
-The following instructions will help you configure Puppetboard and your HTTPD.
+You will need this path in order to configure your HTTPD and WSGI-capable application server.
+
+Please see [an article about more deployment setups here](https://github.com/voxpupuli/puppetboard/docs/Deployment-setups.md).
 
-### Puppet
+## Configuration
 
-Puppetboard is built completely around PuppetDB which means your environment needs to be configured [to do that](https://puppet.com/docs/puppetdb/latest/connect_puppet_master.html#step-2-edit-config-files).
+### PuppetDB
 
-In order to get the reports to show up in Puppetboard you need to configure your environment to store those reports in PuppetDB. Have a look at [the documentation](https://puppet.com/docs/puppetdb/latest/connect_puppet_master.html#edit-puppetconf) about this, specifically the *Enabling report storage* section.
+Of course you need to configure your Puppet Server to store the Puppet run reports in PuppetDB.
+If you haven't done that already please follow the [PuppetDB documentation](https://puppet.com/docs/puppetdb/latest/connect_puppet_server.html)
+about this.
 
-### Settings
+Puppet Enterprise maintains a certificate white-list for which certificates are allowed to access data from PuppetDB.
```

Comment:
This terminology has been updated recently.  See https://puppet.com/docs/puppetdb/latest/release_notes.html#puppetdb-6130

I think the name of the file containing the allowlist has always been configurable, and it's just the configuration option that specifies this file that has been updated.  See https://puppet.com/docs/puppetdb/latest/configure.html#certificate-allowlist

@gdubicki Fantastic work on reworking this README, and I realise that this text is not new, just moved, but would you be able to update this section?

Reply:
Sure @alexjfisher , I haven't noticed that. I replaced this whole fragment as this is not really a Puppet Enterprise-specific feature, at least as of now.

Reply:
Oh, and thank you, @alexjfisher . 😅